README.md

```diff
@@ -93,6 +97,10 @@ ansible-playbook -i ./BraveButtonsConfig/ansible/<inventory file name> \
                  --ask-vault-pass
 ```
 
+1. ssh into the RPi, copy the system_id from `\usr\local\brave\system_id` and add it to the Button Config Google Sheet
+
+1. add an entry for the new hub to the `hubs` table in the db
+
 # How to deploy updates to the RPi fleet using Ansible
```

Comment:
Somewhere in the README, could you please also include instructions on how to limit the command to just one RPi?

Reply:
This pull request isn't the full updated Ansible documentation - I've added your request to that task on ClickUp so I don't forget.